### PR TITLE
New post page

### PR DESCRIPTION
**Pull request by rodrigo-morais**
Adding the functionality to add a new post in the blog.

### CODE REVIEW

**Comment by rodrigo-morais** [comment]
ping @viniciusffj.

File: src/Post/New/Commands.elm [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    textBase64 =
+      case (Base64.encode post.text) of
+        Ok text -> text
+        otherwise -> post.text
+
+    newPosts =
+      List.append posts [{ post | text = textBase64 }]
+
+    postsEncoder =
+      List.map (\post -> postEncoder post) newPosts
```
Comment (viniciusffj) [inline src/Post/New/Commands.elm]: I think you don't need to declare this anonymous func:

``` Elm
List.map postEncoder newPosts
```

File: src/Post/New/Update.elm [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        newPost =
+          case lastPost of
+            Just new -> new
+            otherwise -> model
+
+      in  
+        (newPost, navCmd)
+
+    AddPostFail _ ->
+      (model, Cmd.none)
```
Comment (viniciusffj) [inline src/Post/New/Update.elm:43]: I don't know how much error handling we can do for now, but we could print a message, log or something like that for now.

**Comment by viniciusffj** [comment]
@rodrigo-morais It looks good for me, I just added the following comments ... See if they make send or not.

File: src/Post/New/Commands.elm (hunk 1 above)
Comment (rodrigo-morais) [inline src/Post/New/Commands.elm]: Good call. Thanks! 👍

File: src/Post/New/Update.elm (hunk 2 above)
Comment (rodrigo-morais) [inline src/Post/New/Update.elm:43]: For a while I added a Debug. In the future with style I'll add a visual message with success or error.

**Comment by rodrigo-morais** [comment]
ping @viniciusffj

**Comment by viniciusffj** [comment]
@rodrigo-morais Looks good to me.

**Pull request merged**